pkg/registry/core/trace/common.go

```diff
@@ -44,13 +44,16 @@ func logError(ctx context.Context, err error, operation string) error {
 }
 
 func logObjectTrace(ctx context.Context, k, v interface{}) {
-	s := log.FromContext(ctx)
-	msg := ""
-	cc, err := json.Marshal(v)
-	if err == nil {
-		msg = string(cc)
-	} else {
-		msg = fmt.Sprint(v)
+	registryInfo, ok := trace(ctx)
+	if ok && registryInfo.Request != nil {
```

Comment:
This condition is always `false`

We don't need to check Request and Response, because these fields are used only for `networkservice` (not for `registry`)

Reply:
@glazychev-art , fixes for both comments were made